ui/apps/platform/src/sagas/clusterSagas.js

```diff
@@ -1,21 +0,0 @@
-import { takeLatest, call, fork, put, all } from 'redux-saga/effects';
-
-import * as service from 'services/ClustersService';
-import { actions, types } from 'reducers/clusters';
-
-function* getClusters() {
-    try {
-        const result = yield call(service.fetchClusters);
-        yield put(actions.fetchClusters.success(result.response));
-    } catch (error) {
-        yield put(actions.fetchClusters.failure(error));
-    }
-}
-
-function* watchFetchRequest() {
-    yield takeLatest(types.FETCH_CLUSTERS.REQUEST, getClusters);
-}
-
-export default function* clusters() {
-    yield all([fork(watchFetchRequest)]);
-}
```

Comment:
do we not use this when we're on the `clusters` page?

Reply:
Good question: It calls `fetchClustersWithRetentionInfo` service function.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.js#L123-L134